src/testing/index.ts

```diff
@@ -16,7 +16,9 @@ export { findComponent, niceEl } from './helpers';
 export { By } from '@angular/platform-browser';
 
 // HACKY: export a getter to the current cms-mocks instance
-export const cms: MockHalService = null;
-Object.defineProperty(module.exports, 'cms', {
-  get: () => currentHal() || new MockHalService()
-});
+// export const cms: MockHalService = null;
+// Object.defineProperty(module.exports, 'cms', {
+//   get: () => currentHal() || new MockHalService()
+// });
+// ^ above hack stopped working, so...?
```

Comment:
hack stopped working =/

What this does is get the `cms` service that was provided in the TestBed so we can mock hal api calls off it. I changed it to a function call that usually happens in each test's beforeEach, did the trick.